app/models/pipeline_run_stage.rb

```diff
@@ -168,37 +162,15 @@ def update_job_status
     if failed? || succeeded?
       unless job_log_id
         # set log id if not set
-        _job_status, self.job_log_id, _job_hash, self.job_description = job_info(job_id, id)
+        _job_status, self.job_log_id, self.job_description = job_info(job_id, id)
         save
       end
       return
     end
     # The job appears to be in progress.  Check to make sure it hasn't been killed in AWS.   But not too frequently.
     return unless due_for_aegea_check?
-    self.job_status, self.job_log_id, job_hash, self.job_description = job_info(job_id, id)
-    if [STATUS_ERROR, STATUS_FAILED].include?(job_status)
```

Comment:
Although it looks strange removing this if condition, it is actually redundant with the `unless` statement below. That condition works almost as a "catch all", and does exactly the same thing as this `if` block.